charmverse/app.charmverse.io
# Farcaster login

Test cases:
- login with Farcaster and create an account automatically
- login with Metamask and create an account. Then login with farcaster. It should fail because you already have an account created with metamask
- login with metamask, go to account settings and connect Farcaster
- Disconnect Farcaster
- Use Farcaster as your identity
